aliases/available/general.aliases.bash

```diff
@@ -11,7 +11,7 @@ alias l1='ls -1'
 
 alias _="sudo"
 
-if [ $(uname) = "Linux" ]
+if [ $(uname) = "Linux" ] || brew ls --versions coreutils > /dev/null
```

Comment:
Can't you check for the result of `which ls` or something like that?

I'm asking since I have coreutils installed, but don't use the `gls` command - I use the stock Mac OS `ls` command. This change is going to break the `ls` command for me.

Or here's another idea: Try running `ls --color=auto` and check the exit status of that. If it is `0` you know that the command supports that option.